resources/postgres/resources_dev.go

```diff
@@ -1,9 +0,0 @@
-//go:build !release
```

Comment:
IIRC we ever like to provide a tag to exclude resources if users don't need it. In this patch it's reverted, but why?